torch/_inductor/codegen/cpp.py

```diff
@@ -4734,99 +4994,16 @@ def __post_init__(self):
-
-    def get_root(self):
-        """Get all kernel objects under this loop level"""
-        root = self
-        while root.parent:
-            root = root.parent
-        return root
-
-    def set_kernel(self, kernel: CppKernel):
-        """
-        Set the kernel under this loop level. No split is allowed under
-        this loop level.
-        """
-        if not self.inner:
-            self.kernel = kernel
-            loop: Optional[LoopLevel] = self
-            assert loop is not None
-            return
-        assert len(self.inner) == 1
-        self.inner[0].set_kernel(kernel)
-
-    def get_loops_at(self, depth) -> List["LoopLevel"]:
-        if depth == 0:
-            return [self]
-        else:
-            loops = []
-            for loop in self.inner:
-                loops += loop.get_loops_at(depth - 1)
-            return loops
-
-    def split_with_tiling(self, depth, factor):
-        def clone_inner():
-            inner = []
-            if self.inner:
-                for loop in self.inner:
-                    inner.append(loop.clone())
-            return inner
-
-        def do_split_with_tiling():
-            sympy_factor = sympy.Integer(factor)
-
-            offset = FloorDiv(self.size, sympy_factor) * sympy_factor
-            main_loop = LoopLevel(self.var, offset)
-            main_loop.steps = sympy_factor
-            main_loop.parallel = self.parallel
-            main_loop.collapsed = False
-            main_loop.is_reduction = self.is_reduction
-            main_loop.inner = clone_inner()
-            if main_loop.inner:
-                for loop in main_loop.inner:
-                    loop.parent = main_loop
-
-            tail_loop = LoopLevel(self.var, self.size)
-            tail_loop.offset = offset
-            tail_loop.parallel = self.parallel
-            tail_loop.collapsed = False
-            tail_loop.is_reduction = self.is_reduction
-            tail_loop.inner = clone_inner()
-            if tail_loop.inner:
-                for loop in tail_loop.inner:
-                    loop.parent = tail_loop
-
-            return main_loop, tail_loop
-
-        if depth == 0:
-            main_loop, tail_loop = do_split_with_tiling()
-            parent = self.parent
-            if parent:
-                parent.inner = [main_loop, tail_loop]
-                main_loop.parent = parent
-                tail_loop.parent = parent
-            return main_loop, tail_loop
-        else:
-            assert len(self.inner) == 1
-            return self.inner[0].split_with_tiling(depth - 1, factor)
-
-    def clone(self):
-        loop = copy(self)
-        loop.inner = []
-        if self.inner:
-            for inner_loop in self.inner:
-                inner_loop_clone = inner_loop.clone()
-                inner_loop_clone.parent = loop
-                loop.inner.append(inner_loop_clone)
-        loop.kernel = deepcopy(self.kernel)
+    def split_with_tiling(self, factor):
```

Comment:
"split" is an op that splits the loop into two but this function doesn't seem to do so, right? It seems to create a vectorized loop level?

Reply:
Thanks for reminder here, renamed to `vectorized_with_tiling`.

Reply:
renamed tile.